Makefile

```diff
@@ -33,6 +33,7 @@ KUBE_NETWORK ?= flannel
 
 # Addons
 K8S_DASHBOARD ?= false
+K8S_DASHBOARD_VERSION ?= v1.10.1
```

Comment:
Could you please add it to the https://github.com/galexrt/k8s-vagrant-multi-node#variables section too.
After that LGTM and merge.

Reply:
I have just done it.
Sorry, I should have noticed. Thanks !